sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-redis/pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  -->
```

Comment:
nit: this should be the first line of the file.

Reply:
I locally ran into errors when I put the xml tag after this, saying that "Xml declaration should precede all document content" which is why I placed it this way. Is there a way to address this?